etl/scripts-ccao-data-warehouse-us-east-1/housing/housing-dci_index.py

```diff
@@ -0,0 +1,58 @@
+import os
+import tempfile
+from io import BytesIO
+
+import boto3
+import pandas as pd
+from dotenv import load_dotenv
+
+# Set up the S3 client
+s3 = boto3.client("s3")
+temp_file = tempfile.NamedTemporaryFile(suffix=".xlsx", delete=False)
+
+# Download the file from S3 to your local system
+load_dotenv("etl/.Renviron")
+AWS_S3_RAW_BUCKET = os.getenv("AWS_S3_RAW_BUCKET")[5:]
+file_key = os.path.join("housing", "dci_index", "dci_index.csv")
+
+s3.download_file(AWS_S3_RAW_BUCKET, file_key, temp_file.name)
+
+# Use pandas to read the Excel file, skipping the first two rows
+data = pd.read_csv(temp_file.name)
+
+data = data[data["County"] == "Cook County, Illinois"]
+
+data = data[
```

Comment:
These are the columns relevant to distressed scores. It's probably more than what we want, but I left them all in for the time being.